src/components/AnnouncementModal/AnnouncementModal.jsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { Modal, Button } from 'react-bootstrap';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+
+const StyledModal = styled(Modal)`
+   left: 50%;
+   font-family: 'Nunito', sans-serif;
+`;
+
+const StyledAnnouncements = styled(Modal.Title)`
+   color: #512854;
+   font-weight: 700;
+   font-size: 24px;
+   align: center;
+`;
+const StyledTitle = styled(Modal.Title)`
+   font-weight: 600;
+`;
+
+const StyledSubtitle = styled(Modal.Title)`
+   font-weight: 400;
+   font-size: 18px;
+`;
+
+const StyledBody = styled.p`
+   font-weight: 400;
+   font-style: normal;
+   font-size: 16px;
+   line-height: 21.82px;
+`;
+
+const StyledButton = styled(Button)`
+   variant: secondary;
+   width: 100%;
+   background: #512854;
+`;
+
+const styleDate = (date) => {
+   const year = date.split('-')[0];
+   let month = date.split('-')[1];
+   const day = date.split('-')[2];
+   const months = [
+      null,
+      'January',
+      'February',
+      'March',
+      'April',
+      'May',
+      'June',
+      'July',
+      'August',
+      'September',
+      'October',
+      'November',
+      'December',
+   ];
+   month = months[parseInt(month, 10)];
+   return `${month} ${day}, ${year}`;
+};
+
+const AnnouncementModal = (props) => {
+   const { show, onHide, AnnouncementInfo } = props;
+   console.log('Announcemnt Info');
+   return (
+      <StyledModal style={{ width: '18rem ' }} show={show} backdrop="static">
```

Comment:
This centers the modal.
```suggestion
      <StyledModal centered show={show} backdrop="static">
```